recipes/botorch/meta.yaml

```diff
@@ -0,0 +1,63 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "botorch" %}
+{% set version = "0.2.2" %}
```

Comment:
It looks like the newest version is `0.3.0`. It's ok to start with `0.2.2`, but you'll want to bring it up to date in the feedstock. Are you starting with the older version because you need it?

Reply:
Yes, I will update to the newest versions too. Starting from 0.2.2 because another recipe needs it.